README.md

```diff
@@ -79,15 +79,15 @@ Sailfish OS is a mobile operating system based on open source and developed by t
   * [IRC](https://kiwiirc.com/client/irc.freenode.net:+6697/#Mer-meeting)
 
 * #sailfishos-porters
-  * [Telegram](https://t.me/joinchat/AAAAAAlWDbzGlsfrU5DD6A)
+  * [Telegram](https://t.me/sailfishos_porters)
   * [Matrix](https://riot.im/app/#/room/#freenode_#sailfishos-porters:matrix.org)
   * [IRC](https://kiwiirc.com/client/irc.freenode.net:+6697/#sailfishos-porters)
 * NemoMobile
   * [Telegram](https://t.me/NemoMobile)
   * [Matrix](https://riot.im/app/#/room/#freenode_#nemomobile:matrix.org)
   * [IRC](https://kiwiirc.com/client/irc.freenode.net:+6697/#NemoMobile)
 * SfOS QML App dev
-  * [Telegram](https://t.me/joinchat/Az9rWwlOc_IF0iv8k-3QRg)
+  * Telegram (link unknown)
```

Comment:
Should we actually also remove the entry? Or does the group still exist?

Reply:
I have no idea if it exists or whom to ask about it, but I know it has existed while being silent.

Reply:
Ok, I think we can keep it and hopefully, someone will update it later.